tests/imageIo1.py

```diff
@@ -27,6 +27,7 @@
 Test cases to test image I/O
 """
 import os.path
+import pdb
```

Comment:
why add this? Note that you can run tests with `py.test --pdb` if you want to go into the debugger if a test fails.